### ISSUE TITLE
[Picture-in-Picture] Create new sample that highlights this feature

### ISSUE THREAD

**Issue by CesarValiente** [microsoft/surface-duo-sdk-samples-kotlin#55]
Currently this repository doesn't have a sample that highlights Picture-in-Picture feature.
Since Surface Duo is a dual-screen device, it could be interesting implement a basic sample app that highlights this feature and takes advantage of having two screens side by side.
It could be that are some exceptions that can affect the behavior of Picture-in-Picture and cannot be done (e.g. maybe is not possible to move the small picture to the adjacent screen, so this exercise is also a r&d one).

More info: https://developer.android.com/guide/topics/ui/picture-in-picture

**Comment by EmmanuelDav** [comment]
@CesarValiente is this issue still open?

**Comment by khalp** [comment]
@EmmanuelDav Yes! Since this issue will be an entire sample, if you do decide to work on it, please don't hesitate to ask questions throughout the process.
